Patches/HLX Rimworld UNSC Armoury/Weapons_Ranged.xml

```diff
@@ -0,0 +1,402 @@
+          </Properties>
+          <AmmoUser>
+            <magazineSize>4</magazineSize>
+            <reloadTime>5.6</reloadTime>
+            <ammoSet>AmmoSet_145x114mm</ammoSet>
+          </AmmoUser>
+          <FireModes>
+            <aiAimMode>AimedShot</aiAimMode>
+          </FireModes>
+          <weaponTags Inherit="False">
+            <li>SniperRifle</li>
+            <li>RUA_Gun</li>
+          </weaponTags>
+        </li>
+
+        <!-- === Spacer === -->
+
+        <!-- == Heavy Charge Rifle == -->
+        <li Class="CombatExtended.PatchOperationMakeGunCECompatible">
+          <defName>RUA_Gun_HeavyChargeRifle</defName>
+          <statBases>
+            <WorkToMake>50000</WorkToMake>
+            <SightsEfficiency>1.1</SightsEfficiency>
+            <ShotSpread>0.06</ShotSpread>
+            <SwayFactor>1.52</SwayFactor>
+            <Bulk>11.6</Bulk>
+            <Mass>8</Mass>
+            <RangedWeapon_Cooldown>0.56</RangedWeapon_Cooldown>
+          </statBases>
+          <Properties>
+            <recoilAmount>1.43</recoilAmount>
+            <verbClass>CombatExtended.Verb_ShootCE</verbClass>
+            <hasStandardCommand>true</hasStandardCommand>
+            <defaultProjectile>Bullet_8x35mmCharged</defaultProjectile>
+            <burstShotCount>2</burstShotCount>
+            <ticksBetweenBurstShots>6</ticksBetweenBurstShots>
+            <warmupTime>1.3</warmupTime>
+            <range>62</range>
+            <soundCast>Shot_ChargeRifle</soundCast>
+            <soundCastTail>GunTail_Heavy</soundCastTail>
+            <muzzleFlashScale>11</muzzleFlashScale>
+          </Properties>
+          <AmmoUser>
+            <magazineSize>20</magazineSize>
+            <reloadTime>5.5</reloadTime>
+            <ammoSet>AmmoSet_8x35mmCharged</ammoSet>
+          </AmmoUser>
+          <FireModes>
+            <aiUseBurstMode>true</aiUseBurstMode>
+            <aiAimMode>SuppressFire</aiAimMode>
+          </FireModes>
+          <weaponTags>
+            <li>SpacerGun</li>
+          </weaponTags>
+        </li>
+
+        <!-- == Stanchion Sniper Rifle == -->
+        <li Class="CombatExtended.PatchOperationMakeGunCECompatible">
+          <defName>RUA_Gun_ChargeStanchion</defName>
+          <statBases>
+            <WorkToMake>50000</WorkToMake>
+            <SightsEfficiency>4</SightsEfficiency>
+            <ShotSpread>0.5</ShotSpread>
+            <SwayFactor>1.9</SwayFactor>
+            <Bulk>17</Bulk>
+            <Mass>9</Mass>
+            <RangedWeapon_Cooldown>1.56</RangedWeapon_Cooldown>
+          </statBases>
+          <costList>
+            <Steel>40</Steel>
+            <Plasteel>50</Plasteel>
+            <ComponentSpacer>5</ComponentSpacer>
+          </costList>
+          <Properties>
+            <verbClass>CombatExtended.Verb_ShootCE</verbClass>
+            <hasStandardCommand>true</hasStandardCommand>
+            <defaultProjectile>Bullet_12mmRailgun_Sabot</defaultProjectile>
```

Comment:
I'd look at either the 6mm or 8mm. The 12mm is basically more for vehicle-mounted weapons, almost.

Reply:
I went first with 6mm but it felt that it wasn't doing much damage so i changed to 12mm but maybe you're right and 6mm is the best.